conans/client/conf/config_installer.py

```diff
@@ -1,12 +1,40 @@
 import os
-from conans.tools import unzip
 import shutil
+from six import string_types
+
+try:
```

Comment:
Six exists to avoid this ugly import things. (In addition of many other things)

Reply:
Hey @lasote I've checked that it can be imported with ``future`` 1st option:

http://python-future.org/compatible_idioms.html#urllib-module

but I didn't see any reference in code using future, so I decided to use the ``try/except`` and check in review. I can see that ``future==0.16.0`` is a conan requirement in requirements.txt

Going to use the clean import, thank you!!!